### ISSUE TITLE
Optuna Mode fails. Optuna has migrated to optuna-integration. 

### ISSUE THREAD

**Issue by decmca** [mljar/mljar-supervised#729]
Exception in LightgbmObjective Optuna's integration modules for third-party libraries have started migrating from Optuna itself to a package called `optuna-integration`. The module you are trying to use has already been migrated to `optuna-integration`. Please install the package by running `pip install optuna-integration`.

Is it possible to upgrade mljar-supervised so that Optuna functionality is still available after the implementation of optuna-integration?

**Comment by pplonski** [comment]
Hi @decmca,

Yes, it is possible, we will have intern in the summer, I hope this issue will be fixed. In the meantime, please try to use older Optuna version.

May I ask what is your use case?

**Comment by decmca** [comment]
Yeah i used Optuna 3.5.0, maybe set that as the highest version in requirements.txt for now? 

I'm new here, just testing out AutoML libraries, I think yours is amazing!

**Comment by maciekmalachowski** [comment]
Hi @decmca,
As you asked, I have changed the code so that Optuna functionality is still available.
Among other things, I changed `optuna.integration` to `optuna_integration` in xgboost and lightgbm packages and also changed optuna to optuna-integration in requirements.txt.

Best wishes
